common/changes/@aws/workbench-core-environments/topic-aws-sdk-package-updates_2022-10-14-00-28.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@aws/workbench-core-environments",
+      "comment": "Updated @aws-sdk/* dependencies and locked version at 1.186.0",
```

Comment:
this should be 3.186.0 right? Same for the other change files.

Reply:
Oops, yes. Fixed